### PR TITLE
Migrates tables from tinyint(4) to tinyint(1) for boolean-type content

### PR DESCRIPTION
**Pull request by NiklasBr**
## Additional info  

I prepared this PR, and then I remembered that it had [no effect on storage](https://dev.mysql.com/doc/refman/5.7/en/integer-types.html)/memory or anything like that, it's just a display value as long as it is a `tinyint`, oh well, do what you like with it, at least now it's consistent for all boolean-type columns :)

### CODE REVIEW

**Comment by brusch** [comment]
@NiklasBr  Yes, I think it would make sense to have it consistent across the system. Thanks a lot!

File: bundles/CoreBundle/Migrations/Version20200828091305.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Migrations;
```
Comment (dvesh3) [inline bundles/CoreBundle/Migrations/Version20200828091305.php:3]:
```suggestion
namespace Pimcore\Bundle\CoreBundle\Migrations;
```

Comment (NiklasBr) [reply bundles/CoreBundle/Migrations/Version20200828091305.php]: Thanks!

File: bundles/CoreBundle/Migrations/Version20200828091305.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Log\Handler\ApplicationLoggerDb;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+/**
+ * Class Version20200828091305
+ *
+ * Migrates tables from tinyint(4) to tinyint(1) for boolean-type content
+ *
+ * @package App\Migrations
```
Comment (jdreesen) [inline bundles/CoreBundle/Migrations/Version20200828091305.php:16]:
```suggestion
```

**Comment by dvesh3** [comment]
thank you!

**Pull request merged**